doc/source/basics.rst

```diff
@@ -1776,6 +1783,34 @@ argument:
    s.sort_values()
    s.sort_values(na_position='first')
 
+By Indexes and Values
+~~~~~~~~~~~~~~~~~~~~~
+.. versionadded:: 0.21
```

Comment:
Not sure you need this version-added tag here since you add it below.

Reply:
I left this versionadded tag and removed the one below in the note. This will help people quickly realize that the feature is new and it should be clear that the note below wouldn't apply without the feature.